packages/skeleton-react/src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useMemo } from 'react';
+import { ProgressProps } from './types';
+
+export default function Progress({
+		value,
+		min = 0,
+		max = 100,
+		ariaLabelledby = '',
+		// Root
+		base = 'overflow-x-hidden',
+		bg = 'bg-surface-200-800',
+		height = 'h-2',
+		rounded = 'rounded',
+		classes = '',
+		// Meter
+		meterBase = 'h-full',
+		meterBg = 'bg-primary-500',
+		meterRounded = 'rounded',
+		meterTransition = 'transition-[width]',
+		meterIndeterminateAnim = 'indeterminate',
+		meterClasses = ''
+	}: ProgressProps) {
+
+    useEffect(() => {
+        if (min > max) {
+            console.error(`The minimum value: "${min}" is greater than the maximum value: "${max}"`);
+        }
+    }, [min, max]);
+
+    const indeterminate = useMemo(() => value === undefined, [value]);
+    const fillPercentage = useMemo(() => (indeterminate ? 50 : ((value! - min) / (max - min)) * 100), [value, min, max, indeterminate]);
```

Comment:
Probably also pretty cheap but I think it's fine to leave it in. On the other hand, it's probably going to be the thing that changes the most anyways and will make memoization useless in most cases.

I would tend to leave it in but I'm not 100% convinced. Haven't spent enough time doing low level react optimizations like this.